src/content/docs/validator/amplifier/verifier-onboarding.mdx

```diff
@@ -480,7 +480,7 @@ Prior to running the `ampd` daemon, you will need to set up your wallet with dev
 1. Register support for desired chains, enabling `ampd` to participate in voting and signing for your supported chains.
 
    ```bash
-   ampd register-chain-support amplifier flow
+   ampd register-chain-support validators flow
```

Comment:
There is no more devnet onboarding for verifiers. it's only `validators` in devnet. They go directly to testnet. So this doc either needs to offer tabs or stay `amplifier`.

Reply:
Okay, thanks for clarifying will make this a tabs option instead since we had the question come up so many times want to leave this reference somewhere on the docs.